addons/spreadsheet/static/tests/pivots/model/pivot_positional_formula.test.js

```diff
@@ -315,9 +315,9 @@ test("sort second pivot measure (descending)", async () => {
                 ],
                 model: "partner",
                 rows: [{ fieldName: "product_id" }],
-                sortedColumn: {
-                    groupId: [[], []],
```

Comment:
We should provide a (client-side) upgrade script for such cases, when it's sorted on the total column. Lot's of our dashboards would be broken otherwise